docs/cloud.md

```diff
@@ -0,0 +1,69 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in EKS and AKS.
+
+## Pre-requisites
+1. to run EKS cluster, install and configure aws cli, see
```

Comment:
version 1, version 2, or it doesn't matter?

Reply:
I used version was cli version 1, but I don't know if it matters version 1 or 2.

Reply:
> Su I want to give this a try today. Do you think you can address the comments I left on your original PR (#449), or should I repeat them here?

There is no need to repeat, I will try to address original comments here.
thx, Su